vrk-kpa/suomidigi
# SUOPA-294: CTA Banner text-align variant

To test: 
- `make fresh`
- Go to http://suomidigi.fi.docker.amazee.io/block/add/cta_banner?theme=suomidigi
   - Add **Lohkon kuvaus**: CTA Banneri - Asiantukijapyyntö
   - Add Title, Url and Text for the URL
   - Select text-align position from the select list
   - `Normal` position has question mark, `Right` position is without question mark and text is right aligned
   - Add background image: ![header_illustration](https://user-images.githubusercontent.com/1712902/60449919-5b43a080-9c31-11e9-856f-1961277fe3b2.jpg)
   - Save the block
- Add settings for the block
   - Display title: Uncheck
   - Pages: List wanted pages
   - Language: Finnish
   - Region (alue): Content (sisältö)
   - Save
- Drag the block underneath Main page content and save blocks
- Go to a page you set this block to, scroll to the bottom of the page and make sure the banner is shown (and looks like in proto).

Repeat this in dev-site, or ask me to do it. :)
